tests/conf/l7irish.yaml

```
@@ -0,0 +1,21 @@
+experiment:
+  task: "l7irish"
+  module:
+    loss: "ce"
+    model: "unet"
+    backbone: "resnet18"
+    weights: null
+    learning_rate: 1e-3
+    learning_rate_schedule_patience: 6
+    verbose: false
+    in_channels: 9
+    num_classes: 5
+    num_filters: 1
+    ignore_index: null
```

```suggestion
    ignore_index: 0
```
We want to ignore fill in our calculations. Doesn't matter much for testing, but worth making it realistic.